<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginView</class>
 <widget class="QWidget" name="LoginView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>医院诊疗系统-登录</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_title">
     <property name="text">
      <string>欢迎使用医院诊疗系统</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <bold>true</bold>
      </font>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_form" native="true">
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_username">
        <property name="text">
         <string>用户名：</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEdit_username"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_password">
        <property name="text">
         <string>密码：</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEdit_password">
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_btn" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="pushButton_login">
        <property name="text">
         <string>登录</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_register">
        <property name="text">
         <string>注册</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_dev">
     <property name="text">
      <string>开发者：何文妃 学号：2023414300204</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignBottom</set>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
       <italic>true</italic>
      </font>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
